backend/b2/b2.go

```diff
@@ -662,7 +669,7 @@ var errEndList = errors.New("end list")
 // If hidden is set then it will list the hidden (deleted) files too.
 //
 // if findFile is set it will look for files called (bucket, directory)
-func (f *Fs) list(ctx context.Context, bucket, directory, prefix string, addBucket bool, recurse bool, limit int, hidden bool, findFile bool, fn listFn) error {
+func (f *Fs) list(ctx context.Context, bucket, directory, prefix string, addBucket bool, recurse bool, limit int, hidden bool, versionAt string, findFile bool, fn listFn) error {
```

Comment:
I'd say don't change the arguments for the function since the desired item is in `f` anyway.